certify-mock-identity.properties

```diff
@@ -118,12 +127,99 @@ mosip.certify.key-values={\
-                                  'background_color': '#12107c',\
+                                  'logo': {'url': 'https://${mosip.api.public.host}/inji/mosip-logo.png','alt_text': 'a square logo of a MOSIP'},\
+                                  'background_color': '#f8efea',\
                                   'background_image': { 'uri': 'https://${mosip.api.public.host}/inji/mosip-logo.png' }, \
-                                  'text_color': '#FFFFFF'}},\
+                                  'text_color': '#000000'}},\
                     'order' : {'fullName','phone','dateOfBirth','gender','email','region','province','UIN', 'VID', 'postalCode'}\
-                 }\
+                 },\
+                 "MockVerifiableCredential_mdoc":{\
+                      'format': 'mso_mdoc',\
+                      'doctype': 'org.iso.18013.5.1.mDL',\
+                      'scope' : 'sample_vc_mdoc',\
+                      'cryptographic_binding_methods_supported': {'cose_key'},\
+                      'credential_signing_alg_values_supported': {'ES256'},\
+                      'proof_types_supported': {'jwt': {'proof_signing_alg_values_supported': {'ES256'}}},\
+                      'claims': {\
+                          'org.iso.18013.5.1': {\
+                              'given_name': {'display': {{'name': 'Given Name','locale': 'en'}}},\
+                              'family_name': {'display': {{'name': 'Family Name','locale': 'en'}}},\
+                              'issue_date': {'display': {{'name': 'Issue Date','locale': 'en'}}},\
+                              'expiry_date': {'display': {{'name': 'Expiry Date','locale': 'en'}}},\
+                              'birth_date': {'display': {{'name': 'Birth Date','locale': 'en'}}},\
+                              'issuing_country': {'display': {{'name': 'Issuing Country','locale': 'en'}}},\
+                              'document_number': {'display': {{'name': 'Document Number','locale': 'en'}}}\
+                       }},\
+                      'display': {{'name': 'Mobile Driving License', \
+                                    'locale': 'en', \
+                                    'background_image': { 'uri': 'https://sunbird.org/images/sunbird-logo-new.png' }, \
+                                    'logo': {'url': 'https://sunbird.org/images/sunbird-logo-new.png','alt_text': 'a square logo of a Sunbird Mobile Driving License'},\
+                                    'background_color': '#7C4616',\
+                                    'text_color': '#FDFAF9'}},\
+                       'order' : {'org.iso.18013.5.1~family_name','org.iso.18013.5.1~given_name','org.iso.18013.5.1~document_number','org.iso.18013.5.1~issuing_country','org.iso.18013.5.1~issue_date','org.iso.18013.5.1~expiry_date','org.iso.18013.5.1~birth_date'}\
+              }\
+                }\
+    },\
+  'vd13.1' : {\
```

Comment:
Do we have 13.1 in the current release ?

Reply:
I'll remove it in the next PR @vishwa-vyom

Reply:
Removed vd13.1 from all instances of well-known